### DISCUSSION TITLE
miniconda and anaconda

### DISCUSSION THREAD

**Discussion by scho419** [RoboStack/ros-noetic#256] [Q&A] [answered]
Hi, 

If I already have anaconda installed, is there any reason why I should use miniconda instead of anaconda?

Thanks

**Comment by wolfv** [comment, marked as answer]
I recommend against Anaconda as it comes with too many packages by default. Miniconda or Miniforge are slim base installations.

You should anyways create a new environment. It shoudl be fine to use Anaconda as long as you create a new environment.
